cmd/cluster-olm-operator/main.go

```diff
@@ -1,5 +1,149 @@
 package main
 
+import (
+	"context"
+	goflag "flag"
+	"os"
+
+	"github.com/openshift/library-go/pkg/controller/controllercmd"
+	"github.com/openshift/library-go/pkg/operator/deploymentcontroller"
+	"github.com/openshift/library-go/pkg/operator/staticresourcecontroller"
+	"github.com/openshift/library-go/pkg/operator/status"
+	"github.com/openshift/library-go/pkg/operator/v1helpers"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/apimachinery/pkg/version"
+	"k8s.io/component-base/cli"
+	utilflag "k8s.io/component-base/cli/flag"
+
+	"github.com/openshift/cluster-olm-operator/assets"
+	"github.com/openshift/cluster-olm-operator/pkg/clients"
+)
+
 func main() {
+	pflag.CommandLine.SetNormalizeFunc(utilflag.WordSepNormalizeFunc)
+	pflag.CommandLine.AddGoFlagSet(goflag.CommandLine)
+
+	command := newRootCommand()
+	code := cli.Run(command)
+	os.Exit(code)
+}
+
+func newRootCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "cluster-olm-operator",
+		Short: "OpenShift cluster olm operator",
+	}
+	cmd.AddCommand(newOperatorCommand())
+	return cmd
+}
+
+func newOperatorCommand() *cobra.Command {
+	cmd := controllercmd.NewControllerCommandConfig(
+		"cluster-olm-operator",
+		// TODO: lookup the actual version
+		version.Info{Major: "0", Minor: "0", GitVersion: "0.0.1"},
+		runOperator,
+	).NewCommandWithContext(context.Background())
+	cmd.Use = "operator"
```

Comment:
"start"

Reply:
I thought I saw a few other operators use `operator` as the subcommand. But honestly, `start` seems _much_ better to me. I'll change it.